gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -228,29 +227,6 @@ class BaselineIdea extends AbstractBaselinePlugin {
         return base.appendNode(name, attributes + defaults)
     }
 
-    /**
-     * Configures JDK and Java language level of the given IdeaModel according to the sourceCompatibility property.
-     */
-    private void addJdkVersion(IdeaModel ideaModel) {
-        def compileJavaTask = (JavaCompile) project.tasks.findByName('compileJava')
-        if (compileJavaTask) {
-            def javaVersion = compileJavaTask.sourceCompatibility
-            def jdkVersion = 'JDK_' + javaVersion.replaceAll('\\.', '_')
-            project.logger.debug("BaselineIdea: Configuring IDEA Module for Java version: " + javaVersion)
-
-            if (ideaModel.project != null) {
-                ideaModel.project.languageLevel = javaVersion
```

Comment:
It's probably wrong to set project values here since this is called when applying this plugin to each subproject. If subprojects specify different source compatibilities, then we have a race condition (since I don't think Gradle provides guarantees about the ordering plugins are applied to subprojects).